views/nexthome.tmpl

```diff
@@ -2,11 +2,35 @@
 <!DOCTYPE html>
 <html lang="en">
 {{ template "html-head" "Decred Block Explorer by dcrdata.org"}}
+<script src="https://unpkg.com/tippy.js@2.5.4/dist/tippy.all.min.js"></script>
+<style>
+.block-rows > div > span > a {
+    display: block;
+    width: 100%;
+    height: 100%;
+}
+.container {
+    max-width: 90% !important;
+}
+.flex-space {
+    flex: 0.1 0 10px;
+}
+.tippy-tooltip.light-theme {
+    color: #26323d;
+    box-shadow: 0 0 20px 4px rgba(154,161,177,.15), 0 4px 80px -8px rgba(36,40,47,.25), 0 4px 4px -2px rgba(91,94,105,.15);
+    background-color: #fff;
+}
+.tippy-popper[x-placement^=top] .tippy-arrow {
+    border-top-color: white;
+}
+</style>
+<script type="text/javascript" src="/js/controllers/blocks.js"></script>
+<script type="text/javascript" src="/js/controllers/nexthome-socket-update.js"></script>
```

Comment:
You will need to add `data-turbolinks-eval="false"` to your external js imports. This ensures that your js file is only loaded once and then reused by the browser when needed. i.e.
```html
...
<script type="text/javascript" src="/js/controllers/blocks.js" data-turbolinks-eval="false"></script>
...
```